src/pages/Organization/components/OrganizationLayout.tsx

```diff
@@ -132,8 +139,38 @@ export default function OrganizationLayout({
             ))}
         </Menubar>
       </div>
+      {orgParents.length > 0 && (
```

Comment:
<img width="617" alt="image" src="https://github.com/user-attachments/assets/7469bfe6-8166-465f-9831-cf4316dc1dae" />

Let's change the order, so that breadcrumbs appear before menu bar.